core-services/prow/02_config/_plugins.yaml

```diff
@@ -1486,6 +1486,17 @@ config_updater:
         vsphere:
         - ci
       name: cluster-profile-gcp
+    cluster/test-deploy/ibmcloud/*:
```

Comment:
If the `cluster-profile` files are not needed, then this is not needed too.

Reply:
okay, will remove if we don't use those files.

Reply:
Removed